DTS-STN/dts-rmp
# Nuxt.i18n

# Description
-Installed nuxt i18n framework
-add fr place holder text 
-added a folder structure called Lang for string translation from en and fr
-added a basic header link will be used to be added on the header( no css added yet)
-added browswer detection to determine language used as default french or english

## Test Instructions
Test toggle for lang switch. You will see string changing from english to french
Test browser default lang for language detection when opening the page
    - To do this you have to go into setting and change your default language to french or english

## Checklist
